datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -88,15 +88,19 @@ docker run -i --rm \
 
 Where:
 
-* `<version>`, the version of DTR you're running
-* `<ucp-url>` is the url you use to access UCP
-* `<ucp-username>` is the username of a UCP administrator
-* `<replica-id>` is the ID of the DTR replica to back up
+* `<version>`, the version of DTR you're running.
+* `<ucp-url>` is the url you use to access UCP.
+* `<ucp-username>` is the username of a UCP administrator.
+* `<replica-id>` is the ID of the DTR replica to back up.
+
+You can find the DTR replica IDs, by navigating to the **Stacks** page in
+the **UCP web UI**.
 
+![replica id](../images/backup-and-disaster-recovery-1.png)
 
-This prompts you for the UCP password, backs up the DTR metadata and saves the
-result into a tar archive. You can learn more about the supported flags in
-the [reference documentation](../../reference/cli/backup.md).
+The backup script prompts you for the UCP password, backs up the DTR metadata
+and saves the result into a tar archive. You can learn more about the supported
+flags in the [reference documentation](../../reference/cli/backup.md).
```

Comment:
This might be a good time to recommend using absolute links, so that if you need to move this topic (such as during IA work), they don't break.

Reply:
Since we are versioning content using a directory structure, relative urls require the least amount of work to maintain. When creating a new version we copy `dtr/2.3` into `dtr/2.4` and all the urls for the new version will work.
If we use absolute urls, you need to fix the urls for all the new articles after you copy the directory.